.readthedocs.yml

```diff
@@ -3,7 +3,7 @@ sphinx:
   configuration: docs/conf.py
 formats: all
 python:
-  version: 3.8
+  version: "3.10"
```

Comment:
SUPER clutch catch. I'd hate for that to accidentally normalize as a floating point 3.1 before being interpreted as a version!

Reply:
Yeah, it was a close call, actually! I messed up and amended to fix.

There's a [JSON Schema for readthedocs](https://raw.githubusercontent.com/readthedocs/readthedocs.org/6.3.0/readthedocs/rtd_tests/fixtures/spec/v2/schema.json), so I'll be looking into pulling this in and checking it via `check-jsonschema`.